MekHQ/src/mekhq/gui/FinancesTab.java

```diff
@@ -264,37 +265,37 @@ private CategoryDataset setupMonthlyDataset() {
     	Calendar cal = Calendar.getInstance();
     	
     	String pastMonthYear = "";
-    	long monthlyRevenue = 0;
-    	long monthlyExpenditures = 0;
+    	Money monthlyRevenue = Money.zero();
+        Money monthlyExpenditures = Money.zero();
     	for (int i = 0; i < transactions.size(); i++) {
     		cal.setTime(transactions.get(i).getDate());
     		
     		if (pastMonthYear.equals(df.format(cal.getTime()))) {
-    			if (transactions.get(i).getAmount() > 0) {
-    				monthlyRevenue += transactions.get(i).getAmount();
+    			if (transactions.get(i).getAmount().isPositive()) {
+    				monthlyRevenue = monthlyRevenue.plus(transactions.get(i).getAmount());
     			} else {
-    				monthlyExpenditures += Math.abs(transactions.get(i).getAmount());
+    				monthlyExpenditures = monthlyExpenditures.plus(transactions.get(i).getAmount());
```

Comment:
Missing `Math.abs()`. But couldn't this whole if...else just get rolled into one if you had `getAbsoluteAmount()` on the `Money` class?

Reply:
Not sure how we can roll the if/else together, we are accumulating the results in two different variables and we will need two different string resources later.